Tests/powershell-helpers.ps1

```diff
@@ -41,8 +41,10 @@ function Test-PowershellModuleInstalled($moduleName) {
   }
 }
 
-function Test-CustomVersionOfVagrantDscPluginIsInstalled() {
-  $path = Resolve-Path ~/.vagrant.d/gems/*/gems/vagrant-dsc-*/lib/vagrant-dsc/provisioner.rb
+function Test-CustomVersionOfVagrantDscPluginIsInstalled() {  # does not deal well with machines that have two vesioned folders under /gems/
```

Comment:
This might be better to throw an error instead of just picking the most recent...

Reply:
done